pkg/sfu/streamallocator.go

```diff
@@ -23,6 +23,11 @@ const (
 	NumRequiredEstimatesNonProbe = 8
 	NumRequiredEstimatesProbe    = 3
 
+	NackRatioThresholdNonProbe = 0.06
+	NackRatioThresholdProbe    = 0.04
+
+	NackRatioAttenuator = 0.4 // how much to attenuate NACK ratio while calculating loss adjusted estimate
+
```

Comment:
To explain these constants a bit
- `NackRatioThreshold*` is used to measure if the NACK ratio (ratio of repeatNACKs / packets) exceeds this threshold. If it does, we will declare congestion and adjust the estimate. This helps maybe a few 100 ms in my testing when the estimate is falling. Before the estimate trend detector catches it, this can trigger.
- `NackRatioAttenutor` is used to ratchet down the estimate as a proportion of loss. This is kind of a preemptive/defensive approach to relieve congestion sooner by using a lower estimate which could potentially switch down layers.

Reply:
And the probe phase settings are not more stringent because one of the ideas here is to early abort a probe if it is eventually going to fail.